Makefile

```diff
@@ -229,9 +234,15 @@ copyright:
 	@printf $(COLOR) "Fix license header..."
 	@go run ./cmd/tools/copyright/licensegen.go
 
+goimports:
+	@printf $(COLOR) "Run goimports for modified files..."
+	@printf "Merge base: $(MERGE_BASE)\n"
+	@printf "Modified files: $(MODIFIED_FILES)\n"
+	@goimports -w $(filter %.go, $(MODIFIED_FILES))
```

Comment:
I think we should use golangci-lint here instead of computing MODIFIED_FILES ourselves:
```
@golangci-lint run --no-config --new-from-rev=HEAD~ --whole-files --disable-all -E goimports --fix=true
```